### PR TITLE
MOTECH-2759: Add returned patient UUID bubble to OpenMRS Create Patient Task Action

### PR DESCRIPTION
**Pull request by afijal**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by kmadej** [comment]
test this please

File: openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/DisplayNames.java [hunk 1]
```diff
@@ -43,6 +43,7 @@
     public static final String MOTECH_ID = "openMRS.motechId";
     public static final String LOCATION_FOR_MOTECH_ID = "openMRS.location.name";
     public static final String IDENTIFIERS = "openMRS.patient.identifiers";
+    public static final String UUID = "openMRS.patient.uuid";
```
Comment (martokarski) [inline openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/DisplayNames.java]: Constant with this value exists already.

**Comment by martokarski** [comment]
test this please

**Pull request merged**